src/components/atoms/LogoDismoi.tsx

```diff
@@ -11,12 +11,12 @@ const LogoWrapper = styled.div`
   }
 `;
 
-const Logo = () => {
+const WrappedLogoDismoi = () => {
```

Comment:
Couldn't we have a `Logo(s)` component(s) with the path to the logo as a parameter instead?